eth/vm/forks/istanbul/computation.py

```diff
@@ -35,11 +39,17 @@
 )
 
 
-class IstanbulComputation(ConstantinopleComputation):
+class IstanbulComputation(PetersburgComputation):
     """
     A class for all execution computations in the ``Istanbul`` fork.
-    Inherits from :class:`~eth.vm.forks.constantinople.computation.ConstantinopleComputation`
+    Inherits from :class:`~eth.vm.forks.constantinople.petersburg.PetersburgComputation`
     """
     # Override
     opcodes = ISTANBUL_OPCODES
     _precompiles = ISTANBUL_PRECOMPILES
+
+    def get_gas_meter(self) -> GasMeter:
```

Comment:
I copied this from `ConstantinopleComputation`. Not sure what the right thing to do here is. I have two thoughts in my mind:
(1)  Instead of inheriting from `PetersburgComputation`, inherit from `ConstantinopleComputation`. This could be interpreted as reverting the computation object back to Constantinople. This way `get_gas_meter` won't have to be re-defined.
(2) `Istanbul` is the descendant of `Petersburg` and so it should only inherit from `Petersburg`.  Although (1) is DRY, it might not be semantically correct.

Reply:
I don't have a better solution. This seems better than alternative 1.